### PR TITLE
add some prompt message in undeploy-karmada.sh

### LINKED ISSUE #1042: The Karmada cache was not cleared when executed `hack/undeploy-karmada.sh`
**What happened**:
When I redeploy karmada in my cluster, I firstly undeployed karmada by `hack/undeploy-karmada.sh`, then deployed karmada by `hack/remote-up-karmada.sh`, but the previous resources have been redeployed.

**What you expected to happen**:
I want an initialized environment when I redeploy karmada in my host cluster.

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:
I think that `/var/lib/karmada-etcd` hasn't be deleted when executing `hack/undeploy-karmada-sh`.

Is this on purpose or forgotten to clear？

**Environment**:
- Karmada version: master
- Others:

### PR DESCRIPTION
**Pull request by duanmengkk**
Signed-off-by: duanmeng <duanmeng@cmss.chinamobile.com>

**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
add some prompt message in undeploy-karmada.sh
**Which issue(s) this PR fixes**:
Fixes #1042

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by duanmengkk** [comment]
@lonelyCZ

**Comment by lonelyCZ** [comment]
Test is ok.

```
[root@master67 karmada]# hack/undeploy-karmada.sh ~/.kube/config karmada-host
Switched to context "karmada-host".
namespace "karmada-system" deleted
deleted cluster karmada-apiserver from /root/.kube/config
Property "users.karmada-apiserver" unset.
deleted context karmada-apiserver from /root/.kube/config

Karmada is undeployed successfully.
Please remove the directory '/var/lib/karmada-etcd' in 10.10.103.67 manually to clean the garbage resource thoroughly.
```

Thanks @duanmengkk 


/lgtm

File: hack/undeploy-karmada.sh [hunk 1]
```diff
@@ -39,10 +39,19 @@ rm -fr "${HOME}/.karmada"
...
 kubectl config delete-cluster karmada-apiserver --kubeconfig="${HOST_CLUSTER_KUBECONFIG}"
 kubectl config unset users.karmada-apiserver --kubeconfig="${HOST_CLUSTER_KUBECONFIG}"
 kubectl config delete-context karmada-apiserver --kubeconfig="${HOST_CLUSTER_KUBECONFIG}"
+
+function print_success() {
+  echo
+  echo "Karmada is undeployed successfully."
+  echo "Please remove the directory '/var/lib/karmada-etcd' in $ETCD_HOST_IP manually to clean the garbage resource thoroughly."
+}
+print_success
```
Comment (RainbowMango) [inline hack/undeploy-karmada.sh:57]: Add a new line at the end of file.

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

Thanks.

**Pull request merged**